src/Handler/TaskHandlerFactory.php

```diff
@@ -44,4 +44,14 @@ public function create($className)
 
         return $this->handler[$className];
     }
+
+    /**
+     * Returns all known handler.
```

Comment:
I think it should be handlers in general, shouldn't it? Don't know if it is worth the BC break to rename.